CHANGELOG.md

```diff
@@ -17,7 +23,6 @@
   - This is still considered experimental. Once we have enough feedback we may turn this on by default.
   - Checkout the sample here: https://github.com/getsentry/sentry-java/tree/main/sentry-samples/sentry-samples-spring-boot-webflux-jakarta
   - A new hub is now cloned from the main hub for every request
-- Improve versatility of exception resolver component for Spring with more flexible API for consumers. ([#2577](https://github.com/getsentry/sentry-java/pull/2577))
```

Comment:
Oops, my bad while merging the change from the upstream, sorry.

Thanks for fixing it!

Reply:
No worries, happens to us as well 😅